src/server/apis/dash-data.ts

```diff
@@ -71,59 +72,78 @@ export async function fetchOutgoingData(
     user.name = viewerPrsResult.data.user.name;
     user.avatarUrl = viewerPrsResult.data.user.avatarUrl;
   }
-  const outgoingPrs = [];
+  const outgoingPrs: api.OutgoingPullRequest[] = [];
   if (viewerPrsResult.data.user) {
-    for (const pr of viewerPrsResult.data.user.pullRequests.nodes || []) {
-      if (!pr) {
-        continue;
-      }
+    const requestPrs = viewerPrsResult.data.user.pullRequests.nodes || [];
+    const outgoingPrPromises =
+        requestPrs.map(async(pr): Promise<api.OutgoingPullRequest|null> => {
```

Comment:
Are we sure this runs in parallel?

Reply:
I believe it does yes. Is there a reason you think it's not?